packages/kbn-i18n/src/angular/directive.ts

```diff
@@ -56,12 +58,21 @@ function setHtmlContent(
   $sanitize: (html: string) => string,
   i18n: I18nServiceType
 ) {
-  $element.html(
-    $sanitize(
+  if ($scope.bindValuesAsHtml) {
+    $element.html(
+      $sanitize(
+        i18n($scope.id, {
```

Comment:
__nit:__ please move result of `i18n` call to a separate variable since it's used in both branches.